zhishi-project/Zhishi-Frontend
# #137791351 Add Flag To Prevent Error Reporting In Development Environment

#### What does this PR do?
This PR adds a flag to prevent error reporting when on development environment
#### Description of Task to be completed?
The previous PR for the sentry integration does not include the flag that checks if the app is in production, then allow error reporting.
#### How should this be manually tested?
For this to be tested, an environment variable should be set in env file.
```js
// Set this if app is running on local machine
NODE_ENV='development'

// Set this if app is running on production
NODE_ENV='production'

```
#### Any background context you want to provide?
None.
#### What are the relevant pivotal tracker stories?
#137791351
#### Screenshots (if appropriate)
#### Questions:
